.github/workflows/helium/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+
+node_modules
```

Comment:
please add a new line at the end of the file. By the way, it's better to locate this file in the root directory.

Reply:
Sure, I'll try placing it in the root directory.